<launch>
    <arg name="uav_id" default="0"/>
    <arg name="namespace" value="$(eval 'uav' + str(arg('uav_id')))"/>
	<node pkg="px4_application" type="app_mission_template" name="app_mission_template" output="screen"  ns="$(arg namespace)">
        <rosparam command="load" file="$(find px4_application)/config/mission_config/app_mission_template.yaml" />
        <param name="uav_id" type="int" value="$(arg uav_id)" />
    </node>
</launch>
